src/react-native/21-publishing-and-updating/publishing-and-updating.md

```diff
@@ -10,33 +10,83 @@
 
 In this section, you will:
 
-- Discover the process for publishing an application.
-- Hammer out requirements for publishing on different hosts for apps.
-- Create a new build to update an app.
+- Generate an upload key.
+- Configure Gradle to use the upload key.
+- Sign your AAB with the upload key.
+- Discuss OTA vs AAB updates.
+- How to test new releases.
 
-## Objective 1: Steps for publishing an application
+## Objective 1: Uploading AAB to Google Play store
 
-TODO
+### Generating an upload key
 
-### Concept TODO
+As mentioned in the previous section, if we want to upload an AAB (Android App Bundle) to the Google Play store, we need to generate an upload key and use it to sign the AAB. The instructions for generating an upload key differ between Windows and macOS, they can be found [here](https://reactnative.dev/docs/signed-apk-android).
```

Comment:
In general, try to [avoid generic phrases](https://webaim.org/techniques/hypertext/link_text#uninformative) like “here” for links.

Reply:
got it, will rephrase all the [here] in this page 😄